Dockerfile

```diff
@@ -2,10 +2,6 @@ FROM python:3.7-alpine
 
 ENV PYTHONUNBUFFERED 1
 
-# TODO: Install build deps?
-RUN apk add git
-RUN apk add openssh
-
```

Comment:
I don't think we need these on our image now that we are using the circleci image on their platform, yah?